Common/Product/TestAdapter/VisualStudioApp.cs

```diff
@@ -75,6 +67,10 @@ class VisualStudioApp : IDisposable {
         [DllImport("ole32.dll")]
         private static extern int CreateBindCtx(uint reserved, out IBindCtx ppbc);
 
+        public DTE GetDTE() {
+            return GetDTE(_processId);

```

Comment:
I believe this can return null - can we check here and get a better exception than a nullref later on? InvalidOp is fine, or make a new one so that we can at least capture the name in crash reports.